scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/TextWriter.java

```diff
@@ -29,7 +29,7 @@ public TextWriter(HuginnConfiguration config) {
   /**
```

Comment:
:large_orange_diamond: Are we using/plan to use this TextWriter class for anything? If not I vote that it be removed.

Reply:
I do not know so we can remove it.

Reply:
Great. Let's remove it.